test/web3-functions/base/governance-executor.test.ts

```diff
@@ -0,0 +1,225 @@
+import hre from 'hardhat'
+import chai from 'chai'
+import { Contract, ContractFactory } from '@ethersproject/contracts'
+import { Web3FunctionHardhat } from '@gelatonetwork/web3-functions-sdk/hardhat-plugin'
+import { Web3FunctionResultCallData } from '@gelatonetwork/web3-functions-sdk/*'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import {
+    SnapshotRestorer,
+    impersonateAccount,
+    mine,
+    setBalance,
+    takeSnapshot,
+} from '@nomicfoundation/hardhat-network-helpers'
+import { baseGovernanceExecutorAbi } from '../../../abis'
+import { addresses } from '../../../utils'
+
+const { w3f, ethers } = hre
+const { expect } = chai
+
+describe('GovernanceExecutor', function () {
+    this.timeout(0)
```

Comment:
we want to ship it like this? if yes we should have a global timeout in CI pipeline config.